examples/window/minimising.rs

```diff
@@ -0,0 +1,75 @@
+//! A test to confirm that `bevy` allows minimising the window
+//! This is run in CI to ensure that this doens't regress again.
+use bevy::prelude::*;
+
+fn main() {
+    // TODO: Combine this with `resizing` once multiple_windows is simpler than
+    // it is currently.
+    App::new()
+        .insert_resource(WindowDescriptor {
+            title: "Minimising".into(),
+            ..Default::default()
+        })
+        .add_plugins(DefaultPlugins)
+        .add_system(minimise_automatically)
+        .add_startup_system(setup_3d)
+        .add_startup_system(setup_rect)
```

Comment:
I think calling this `setup_2d` is clearer then calling it like the example it's taken from.

```suggestion
        .add_startup_system(setup_2d)
```